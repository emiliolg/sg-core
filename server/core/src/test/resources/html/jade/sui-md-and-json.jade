doctype html
html( sg-view="tekgenesis.service.html.Views", sg-params="product : tekgenesis.samples.service.Product")
    head
        title= 'Markdown and JSON examples'
    body
        h2= 'Markdown to html'
        div
            :md
                <!-- Comment -->

                ### Some Sub Title

                - With important
                - Bullets

        h2= 'Parameter to JSON:'
        script
            :plain
                var myProductJson =
            :json
                product