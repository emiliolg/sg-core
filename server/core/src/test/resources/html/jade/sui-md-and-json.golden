<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="product : tekgenesis.samples.service.Product">
  <head>
    <title>Markdown and JSON examples</title>
  </head>
  <body>
    <h2>Markdown to html</h2>
    <div><!-- Comment -->
<h3><a href="#some-sub-title" name="some-sub-title">Some Sub Title</a></h3>
<ul>
  <li>With important</li>
  <li>Bullets</li>
</ul>
    </div>
    <h2>Parameter to JSON:</h2>
    <script>var myProductJson ={"productId":"product.id","model":"Some model","description":"Some description with \"quoted\" words.","price":89.99,"created":"2014-01-31T03:00:00Z"}
    </script>
  </body>
</html>